### DISCUSSION TITLE
Question 89: Default values in Phenology tab? [WCG]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#1175] [Q&A]
Description: I can’t figure out how to set a default value in the new Phenology tab. I want to put “Dried specimen” in PrePrepType_tab, but none of the available options in the Set-Defaults menu seem to refer to this field. 

(I assume having a default here is desirable because all 99%+ of our records with ObjKindOfObject=”Dried specimen” got that value in Phenology during migration)

**Comment by rondlg** [comment]
Response: On a new record, enter the value “Dried specimen” in the Prep field and use the “Set as Default” option to create your default settings.

![1175](https://github.com/user-attachments/assets/5f8bfe60-dda3-4ed6-b4c9-44a082dc807e)

**Reply by rondlg** [reply]
@fmjjones Requested that a zendesk ticket be submitted

**Comment by fmjjones** [comment]
ZenDesk ticket[74098](https://fieldmuseum.zendesk.com/agent/tickets/74098). Issue resolved.
